src/client/src/ui/analytics/components/AnalyticsLineChart.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import moment from 'moment'
+import numeral from 'numeral'
+import { XAxis, YAxis, ResponsiveContainer, LineChart, ReferenceLine, Tooltip, Line } from 'recharts'
+import { ToolTipStyle, ToolTipChildRight, ToolTipChildLeft, AnalyticsLineChartStyle } from './styled'
+import { AnalyticsLineChartModel } from '../model/AnalyticsLineChartModel'
+
+interface LineChartProps {
+  model: AnalyticsLineChartModel
+}
+
+interface DataPoint {
+  x: string
+  y: string
+}
+
+const tickFormatYAxis: ((x: string) => string) = x => numeral(x).format('0.0a')
+
+const getLinearGradientOffset: ((data: DataPoint[]) => number) = data => {
+  const yValues = data.map(i => parseInt(i.y, 10))
+  const dataMax = Math.max(...yValues)
+  const dataMin = Math.min(...yValues)
+
+  if (dataMax <= 0) {
+    return 0
+  }
+  if (dataMin >= 0) {
+    return 1
+  }
+
+  return dataMax / (dataMax - dataMin)
+}
+
+interface ToolTipProps {
+  payload: any[]
+  label: string
+}
+
+const CustomTooltip: React.SFC<ToolTipProps> = ({ payload, label }) => {
+  return (
+    <ToolTipStyle>
+      <ToolTipChildLeft>{label}</ToolTipChildLeft>
+      <ToolTipChildRight>{payload.length > 0 && numeral(payload[0].value).format('0.0a')}</ToolTipChildRight>
+    </ToolTipStyle>
+  )
+}
+
+interface LineChartState {
+  offset: number
+}
+class LineCharts extends React.PureComponent<LineChartProps, LineChartState> {
+  private intervalWidth: number = 30
```

Comment:
Perhaps this is a not a requirement right now, but the width of the interval probably should change depending on the window width.